docs/preview-builds.md

```diff
@@ -0,0 +1,33 @@
+# Testing monorepo changes against other projects
+
+Let's say that you're working on a feature in one of our products and this feature relies on changes that you need to make to one or more packages within this repo. You make those changes here, but now you want to make sure that they fulfill the need you have in your product and don't break anything. How do you integrate these changes into your product?
+
+Previously you might have used the "commit id" method to solve this problem, where you:
+
+- push up a branch in this repo
+- copy the commit id of the latest commit on that branch
+- open `package.json` in your project
+- replace the version of some package with `MetaMask/controllers#<commit id>`
+
+That method won't work anymore (at least until we have all repos using Yarn v3). So now what do you do?
+
+Enter **preview builds**. These are ephemeral versions of packages which are built on demand and published to GitHub Package Registry.
+
+To use them, you'll need to do a few of things:
+
+1. You'll need to create a personal access token within your GitHub account. If you have not already done so, go into the [token settings for your account](https://github.com/settings/tokens) and [create a token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic). (Note: there are two types of tokens; you'll want a **classic** token.)
```

Comment:
I think we can assume the reader knows how to get a GH access token (or look it up elsewhere)? Emphasis on classic seems relevant, though.

Reply:
The linked article is the official documentation. That does seem like the best place to point people. I guess we could shorten this sentence to omit the "go into the token settings" part, as that's already covered by the article as well.

Reply:
Something like this?
```suggestion
1. You'll need to create a **classic** [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic).
```
alternatively,
```suggestion
1. You'll need to [create](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic) a **classic** personal access token.
```